optax/_src/transform_test.py

```diff
@@ -202,6 +202,16 @@ def test_centralize(self, inputs, outputs):
     centralized_inputs, _ = centralizer.update(inputs, None)
     chex.assert_tree_all_close(centralized_inputs, outputs)
 
+  def test_sherman_morrisson(self):
+    x = 1
```

Comment:
This is a very special value: in line 210 it will mean that `x` doesn't have an effect. Could you set up the test such that `x` is used in a non-trivial way?